### ISSUE TITLE
TF-slim inceptionv3 

### ISSUE THREAD

**Issue by rickymf4** [tensorflow/models#996]
## TF-slim inceptionv3 train from scratch

I use slim/train_image_classifier.py to train a inception_v3 model on my own dataset:
`
python train_image_classifier.py 
    --train_dir=${TRAIN_DIR} 
    --dataset_name=mydataset 
    --dataset_split_name=train 
    --dataset_dir=${DATASET_DIR} 
    --model_name=inception_v3 
    --num_clones=2
`

the loss curve was strange as the tensorboard shown, it was a linearly decreasing straight line with a little bump part.
![image](https://cloud.githubusercontent.com/assets/9128530/22721137/ea6470e4-ede9-11e6-9a6b-afd4e37315e4.png)

the following are the last outputs, the loss was decreased  0.0001 every 20 or 30 steps：
`
    
   INFO:tensorflow:global step 34590: loss = 0.5359 (1.17 sec/step)
   INFO:tensorflow:global step 34600: loss = 0.5358 (1.15 sec/step)
   INFO:tensorflow:global step 34610: loss = 0.5358 (1.17 sec/step)
   INFO:tensorflow:global step 34620: loss = 0.5357 (1.12 sec/step)
   INFO:tensorflow:global step 34630: loss = 0.5357 (1.16 sec/step)
   INFO:tensorflow:global step 34640: loss = 0.5356 (1.16 sec/step)
   INFO:tensorflow:global step 34650: loss = 0.5356 (1.16 sec/step)
   INFO:tensorflow:global step 34660: loss = 0.5355 (1.15 sec/step)
   INFO:tensorflow:global step 34670: loss = 0.5355 (1.15 sec/step)
   INFO:tensorflow:global step 34680: loss = 0.5355 (1.18 sec/step)
   INFO:tensorflow:global step 34690: loss = 0.5354 (1.17 sec/step)
   INFO:tensorflow:global step 34700: loss = 0.5354 (1.15 sec/step)
   INFO:tensorflow:global step 34710: loss = 0.5353 (1.15 sec/step)
   INFO:tensorflow:global step 34720: loss = 0.5353 (2.25 sec/step)
   INFO:tensorflow:global step 34730: loss = 0.5353 (2.22 sec/step)
   INFO:tensorflow:global step 34740: loss = 0.5352 (1.16 sec/step)
   INFO:tensorflow:global step 34750: loss = 0.5352 (1.16 sec/step)
   INFO:tensorflow:global step 34760: loss = 0.5351 (1.18 sec/step)
   INFO:tensorflow:global step 34770: loss = 0.5351 (1.15 sec/step)
   INFO:tensorflow:global step 34780: loss = 0.5350 (1.17 sec/step)
   INFO:tensorflow:global step 34790: loss = 0.5350 (1.15 sec/step)
   INFO:tensorflow:global step 34800: loss = 0.5349 (1.12 sec/step)
   INFO:tensorflow:global step 34810: loss = 0.5349 (1.12 sec/step)
   INFO:tensorflow:global step 34820: loss = 0.5349 (1.16 sec/step)
   INFO:tensorflow:global step 34830: loss = 0.5348 (1.16 sec/step)
   INFO:tensorflow:global step 34840: loss = 0.5348 (1.18 sec/step)
   INFO:tensorflow:global step 34850: loss = 0.5347 (1.12 sec/step)
   INFO:tensorflow:global step 34860: loss = 0.5347 (1.12 sec/step)
   INFO:tensorflow:global step 34870: loss = 0.5347 (1.18 sec/step)
   INFO:tensorflow:global step 34880: loss = 0.5346 (1.13 sec/step)
   INFO:tensorflow:global step 34890: loss = 0.5346 (1.18 sec/step)
   INFO:tensorflow:global step 34900: loss = 0.5345 (1.16 sec/step)
   INFO:tensorflow:global step 34910: loss = 0.5345 (1.15 sec/step)
   INFO:tensorflow:global step 34920: loss = 0.5344 (1.17 sec/step)
   INFO:tensorflow:global step 34930: loss = 0.5344 (1.14 sec/step)
   INFO:tensorflow:global step 34940: loss = 0.5344 (1.15 sec/step)
   INFO:tensorflow:global step 34950: loss = 0.5343 (1.14 sec/step)
   INFO:tensorflow:global step 34960: loss = 0.5343 (1.17 sec/step)

` 

mydataset.py is the same as flowers.py except:
`
 SPLITS_TO_SIZES = {'train': 18000000, 'validation': 400000}

 _NUM_CLASSES = 4
`
 
Is it normal?  Thanks for any help.

**Comment by girving** [comment]
Questions like this belong on StackOverflow.  Github issues are for bug reports and feature requests.
